programs/perpetuals/src/error.rs

```diff
@@ -48,4 +48,9 @@ pub enum PerpetualsError {
     UnsupportedToken,
     #[msg("Instruction is not allowed at this time")]
     InstructionNotAllowed,
+    // Test errors
+    #[msg("Instruction data is too short.")]
+    InstructionDataTooShort,
```

Comment:
are InstructionDataTooShort and InstructionDataSliceMisaligned used somehow? I can't find it